loadRTs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+curl -i -X PUT --data-binary @fixtures/base-container.jsonld -H "Content-Type:application/ld+json" http://localhost:8080/
+
+sleep 1
+echo ""
+curl -i -H "Accept:application/ld+json" http://localhost:8080/
+
+sleep 1
+echo ""
+curl -i -X PUT --data-binary @fixtures/repository-container.jsonld -H 'Link: <http://www.w3.org/ns/ldp#BasicContainer>; rel="type"' -H "Content-Type:application/ld+json" http://localhost:8080/repository
+
+sleep 1
+echo ""
+curl -i -H "Accept:application/ld+json" http://localhost:8080/repository
+
+sleep 1
+echo ""
+curl -i -X POST --data-binary @fixtures/group_defs/ld4p.jsonld -H 'Link: <http://www.w3.org/ns/ldp#BasicContainer>; rel="type"' -H "Content-Type:application/ld+json" -H "Slug:ld4p" http://localhost:8080/repository -u cmharlow:S3cr3t!
+
```

Comment:
um ... where does "repository" come from?  Does that have to do with non-rdf content?

I've been under the impression the groups will always be directly under the root container, perhaps from   https://github.com/LD4P/sinopia_server/wiki/Draft-Notes-for-Sinopia-Server-API-Spec

Reply:
the wiki does currently use `/` as the base container, but that's actually in need of fixing, the plan since at least late last year has been to use `/repository` as the base container for everything.  i guess maybe the wiki entry (or the notes that were used to create it) preceded that decision?  i'm not sure if this is documented anywhere other than in the swagger spec (but it was introduced in [this PR](https://github.com/LD4P/sinopia_server/pull/21), since all the examples i'd seen up to that point when pairing with christina and jeremy used `/repository`).

looking at the swagger spec history, christina did originally have everything under the base container, then under `/resources`, then i moved it to `/repository`, but that was just me doing the grunt work after asking what base path we wanted, not deciding myself on a new path.
https://github.com/LD4P/sinopia_server/commit/b45c3e4fe0b811ee51bb81ec1b06d3c376a5753c#diff-bac86ae86657c2fa2c3dd2ab26df4960
https://github.com/LD4P/sinopia_server/commit/35751c59434c189ae7dc77914cd9040f00e76470#diff-bac86ae86657c2fa2c3dd2ab26df4960

@jermnelson, correct me if i'm wrong, i think i ran that by you at some point again in the last few weeks.

Reply:
This is at least what I have been hearing, and matches my understanding, @jmartin-sul

Reply:
Hi @jmartin-sul, that is what I recall as well; we would use `/repository` as the base container.

Reply:
👍 excellent.  thanks for the info.  mebbe this should be documented somewhere?  Ticket to add it to github pages, mebbe?